lib/SIL/SILGlobalVariable.cpp

```diff
@@ -106,6 +106,7 @@ bool SILGlobalVariable::isValidStaticInitializerInst(const SILInstruction *I,
     }
     case SILInstructionKind::StringLiteralInst:
       switch (cast<StringLiteralInst>(I)->getEncoding()) {
+        case StringLiteralInst::Encoding::Bytes:
```

Comment:
Can you add a test for this?

Reply:
Same question as above.

Reply:
I think that this is reasonable to not add a test for. The other one I think is more suspect.